phpunit.xml.dist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file about anyways?

Reply:
It just allows you to run `phpunit` in the root without having to specify the location of the tests. It's best practice to have one! I didn't realize I added one in this PR though, so I can remove it :P

Reply:
eh its fine! Keep it.